source/docs/blockchain_basic_configuration.md

```diff
@@ -0,0 +1,66 @@
+title: Blockchain configuration
+---
+
+When in development phase, it is necessary to use a local Ethereum node to deploy our contracts to, either using a simulator or client like Geth or Parity.
+
+To set this node, Embark looks at `blockchain.js`, which by default is at `config/blockchain.js`.
+
+It is a complex file, but Embark, when using the `embark new` and `embark demo` command, creates a default `blockchain.js` file for you that contains all the basic parameters for you to start a Geth node or a Ganache simulator.
```

Comment:
Tiny tip: Generally it's better to make things sounds positive instead of negative. So instead of telling people that the file is complex, let's just focus on that Embark luckily comes with decent defaults out of the box, so we don't have to worry too much.

So maybe something like

> Embark offers a lot of configuration options and most of them already come with a decent default so we can start right away.

Can definitely worded better..